parser_library/test/expressions/logical_expression_test.cpp

```diff
@@ -170,6 +174,7 @@ TEST(logical_expressions, priority)
 &A4 SETB (1 XOR 0 OR 1) 0
 &A5 SETB (1 AND 0 OR 1 AND 1)  
 &A6 SETB (NOT 0 AND NOT 0 AND 50 GT 126+4)
+&A7 SETB (NOT 0 OR NOT 0)
```

Comment:
add `&A8 SETB (NOT 0 EQ NOT 0)` == 1

Reply:
It's contained in `relational_operator_eq` tests but I can add it here anyway.